src/national-parks.js

```diff
@@ -440,6 +502,8 @@ const nationalParks = [
    "latitude": "20.72°N",
    "longitude": "156.17°W",
    "annualVisitors": 1112390,
+   "petsWelcome": true,
+   "editorsChoice": "Hosmer's Grove Nature Trail",

```

Comment:
Add \\ before single quotes

Reply:
Nevermind - since JSON only recognizes double quotes, we don't need to escape a single quote.